BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -617,6 +684,12 @@ - (NSURLSessionTask *)signInWithCredential:(NSString *)credentialKey value:(id<S
         return nil;
     }
     
+    // If an auth/reauth call is already in progress, just queue up the completion handler and return the
+    // existing NSURLSessionTask object.
+    if ([self testAndSetAuthCallInProgressWithCompletion:completion]) {
+        return self.inProgressAuthURLSessionTask;
```

Comment:
Does the access to the session task also need to be synchronous?

Reply:
No, because access to the flag returned by testAndSetAuthCall...etc. is serialized so only one thread (the one that actually flipped it from "no" to "yes") can ever get past this to eventually change its value.

On the other hand, it turns out I'm never actually setting this value anywhere... oops.

Reply:
Actually on second thought, I do need to make sure that one thread can't clear it in between the test-and-set and this return. Will fix that. 👍

Reply:
I ended up just removing the inProgressAuthURLSessionTask property because it's not really needed, and it probably makes sense for only the "original" auth/reauth call that kicked off the Bridge request to have access to the session task anyway.